src/style.css

```diff
@@ -1 +1,140 @@
-/* add styles here */
+/* #region reset styles */
+html,
+body {
+  margin: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+input {
+  font-family: inherit;
+  font-size: inherit;
+  color: inherit;
+  background-color: transparent;
+}
+
+input::-ms-clear {
+  display: none;
+}
+
+input[type='submit'] {
+  display: inline-block;
+  box-shadow: none;
+  background: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+input:focus,
+input:active {
+  outline: none;
+}
+
+/* #endregion */
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
```

Comment:
The font files `Avenir-Book.ttf` and `Avenir-Heavy.ttf` are referenced here but are missing from the project. Ensure these files are included in the `fonts` directory to avoid font loading issues.